launchable/commands/subset.py

```diff
@@ -117,14 +117,26 @@ def separator(self, s: str):
             self._separator = s
 
         def test_path(self, path: TestPathLike):
+            def rel_base_path(path: str):
+                p = path
```

Comment:
You don't need to copy the string because it's an immutable value. You can simply write it like the following code.

```python3
return normpath(relpath(path, start=base_path)) if base_path else path
```